pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -522,11 +532,22 @@ func (tc *TidbCluster) TiKVIsAvailable() bool {
 		}
 	}
 
+	var peerAvailableNum int32
+	for _, store := range tc.Status.TiKV.PeerStores {
+		// filter out the TiFlash PeerStores
```

Comment:
`TiKV.PeerStores` should not include tiflash stores.

Reply:
I will submit a new PR for this.

Reply:
Will you remove this check in https://github.com/pingcap/tidb-operator/pull/3527?

Reply:
This check(in this PR) could be removed if 3527 is merged.